### ISSUE TITLE
Build error missing version.rs

### ISSUE THREAD

**Issue by 0xpr03** [johnschug/rust-gpg-error#4]
I get the following error:
```
error: couldn't read /home/user/Dev/test/target/debug/build/libgpg-error-sys-ca1f23b870971a25/out/version.rs: No such file or directory (os error 2)                                                                  
 --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/libgpg-error-sys-0.5.0/lib.rs:7:1                    
  |                                                                                                                  
7 | include!(concat!(env!("OUT_DIR"), "/version.rs"));                                                               
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                               
                                                                                                                     
error: aborting due to previous error
                                                                                                                     
error: Could not compile `libgpg-error-sys`.
```

Ubuntu 18.04, with libgpg-error-dev and libgpgme11-dev.

**Comment by nlfiedler** [comment]
Strangely enough, it builds fine on Debian buster, which makes you wonder.

**Comment by johnschug** [comment]
This should be fixed in libgpg-error-sys v0.5.1. For some reason the format of the version string output by the configuration script (i.e., `gpg-error-config --version`) changed, and the build script was unable to parse the expanded format. 

It used to output a string like `a.b` or `a.b.c`. Now it sometimes outputs a patch level as well (e.g.,
`a.b.c-patch`).
